bundles/org.openhab.binding.androiddebugbridge/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="androiddebugbridge" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>AndroidDebugBridge Binding</name>
+	<description>This is the binding for connect to Android devices using the Android Debug Bridge protocol.</description>
+	<author>Miguel Álvarez</author>
```

Comment:
The author tag is deprecated and should therefore be removed. See openhab/openhab-core#1844.